src/evm-block-extractor/src_resources/db/postgres/migrations/00004_add_deleted_blocks.sql

```diff
@@ -0,0 +1,30 @@
+
+-----------------------------
+-- Begin - DISCARDED_EVM_BLOCK -
+-----------------------------
+
+create table DISCARDED_EVM_BLOCK (
+    ID bigint primary key,
+    DATA JSONB,
+    DISCARDED_AT TIMESTAMP,
+    REASON TEXT 
+);
+
+CREATE INDEX DISCARDED_EVM_BLOCK_INDEX_DISCARDED_AT ON DISCARDED_EVM_BLOCK( DISCARDED_AT );
+
+-- End - DISCARDED_EVM_BLOCK -
+
+-----------------------------------------
+-- Begin - DISCARDED_EVM_TRANSACTION -
+-----------------------------------------
+
+create table DISCARDED_EVM_TRANSACTION (
```

Comment:
Since the block is saved in the JSON format, you can have all the TXs inside the block itself instead of storing them in a separate table

Reply:
I thought about it. Using two separate tables allows to perform blocks and txs movement atomically in one DB transaction.

Reply:
Do you mean in one query?

Reply:
Ok, I'll try to make it into single table.

Reply:
Done